modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -27,3 +27,17 @@ serviceNetwork:
 ====
 The API VIP IP address and the Ingress VIP address must be of the primary IP address family when using dual-stack networking. Currently, Red Hat does not support dual-stack VIPs or dual-stack networking with IPv6 as the primary IP address family. However, Red Hat does support dual-stack networking with IPv4 as the primary IP address family. Therefore, the IPv4 entries must go *before* the IPv6 entries.
 ====
+
+To provide an interface to the cluster for applications that use IPv4 and IPv6 addresses, configure IPv4 and IPv6 virtual IP (VIP) address endpoints for the Ingress VIP and API VIP services. Edit the `apiVIPs` and `ingressVIPs` configuration settings in the `install-config.yaml` file to configure IPv4 and IPv6 address endpoints. The `apiVIPs` and `ingressVIPs` configuration settings use a list format. The order of the list indicates the primary and secondary VIP address for each service.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    apiVIPs:
+      - <api_ipv4>
+      - <api_ipv6>
+    ingressVIPs:
+      - <wildcard_ipv4>
+      - <wildcard_ipv6>
+----
```

Comment:
@cybertron - FYI - mentioned ipv4/6 in this placeholder text in order to further convey ipv4 should be the primary address